vmngclient/utils/feature_template.py

```diff
@@ -0,0 +1,67 @@
+from typing import Callable, Dict, Optional, Union
+
+from vmngclient.api.templates.models.supported import FeatureTemplateType, supported_models
+from vmngclient.exceptions import TemplateTypeError
+
+
+def choose_model(type_value: Union[str, FeatureTemplateType]) -> Callable:
```

Comment:
If we want to pick a model class by template name identifier why don't use a dict? (see my comment supported.py)
Also return type seems wrong.